Casks/v/valkyrie.rb

```diff
@@ -1,16 +1,16 @@
 cask "valkyrie" do
-  version "2.5.7"
-  sha256 "e667934e6ee3f27d3f81591b830d2be4513c8997f54f5312be56917c0697ce75"
+  version "2.6.0"
+  sha256 "0aaa7435012fdae6f39b88f7cbbc9a179c992acd068d4ae6539fd973e0aa17a6"
 
-  url "https://github.com/NPBruce/valkyrie/releases/download/release/#{version.major_minor}/v#{version}/valkyrie-macos-#{version}.tar.gz",
+  url "https://github.com/NPBruce/valkyrie/releases/download/release/#{version.major_minor}/#{version}/valkyrie-macos-#{version}.tar.gz",
       verified: "github.com/NPBruce/valkyrie/"
   name "Valkyrie"
   desc "Game Master for Fantasy Flight board games"
   homepage "https://npbruce.github.io/valkyrie/"
 
   livecheck do
     url :url
-    regex(%r{/v(\d+(?:\.\d+)+)$}i)
+    regex(%r{/(\d+(?:\.\d+)+)$}i)
```

Comment:
Let's do this in case it's back in the next version
```suggestion
    regex(%r{/v?(\d+(?:\.\d+)+)$}i)
```

Reply:
That seems fine.

What is the best way to handle the 'v' being present or absent in the download url?

https://github.com/NPBruce/valkyrie/releases/download/release/2.5/v2.5.7/valkyrie-macos-2.5.7.tar.gz
https://github.com/NPBruce/valkyrie/releases/download/release/2.6/2.6.0/valkyrie-macos-2.6.0.tar.gz